pkg/ttnpb/field_mask_validation.go

```diff
@@ -866,6 +866,12 @@ var AllowedFieldMaskPathsForRPC = map[string][]string{
 	"/ttn.lorawan.v3.UserRegistry/List":                omitFields(UserFieldPathsNested, "password", "temporary_password"),
 	"/ttn.lorawan.v3.UserRegistry/Update":              omitFields(UserFieldPathsNested, "password", "password_updated_at"),
 	"/ttn.lorawan.v3.EntityRegistrySearch/SearchUsers": omitFields(UserFieldPathsNested, "password", "temporary_password"),
+
+	// Device Repository:
+	"/ttn.lorawan.v3.DeviceRepository/GetBrand":   EndDeviceBrandFieldPathsNested,
+	"/ttn.lorawan.v3.DeviceRepository/ListBrands": EndDeviceBrandFieldPathsNested,
+	"/ttn.lorawan.v3.DeviceRepository/GetModel":   EndDeviceModelFieldPathsNested,
+	"/ttn.lorawan.v3.DeviceRepository/ListModels": EndDeviceModelFieldPathsNested,
```

Comment:
We don't need the others?

Reply:
The other RPCs do not accept field mask paths.